css/main.css

```diff
@@ -75,7 +75,40 @@ body {
 /* ––––––––– INTRO ––––––––––––– */
 
 .intro {
+  padding-top: 174px;
+  text-align: center;
+}
+
+.intro-title {
+  font-weight: var(--fw-bold);
+  font-size: 64px;
+  line-height: 96px;
+  color: var(--c-main-body);
+}
+
+.intro-subtitle {
+  font-size: 24px;
+  line-height: 59px;
+  font-weight: var(--fw-regular);
+  color: var(--c-main-body);
+}
+
+.intro-buttons {
+  display: flex;
+  justify-content: center;
+  padding-top: 30px;
```

Comment:
Please use margins to add some distance between blocks that lay next to each other